docs/mdbook/default.nix

```diff
@@ -67,12 +67,13 @@ let
           filterAttrs (_: component: component.isOption && (isVisible component)) opts;
       path = removeWhitespace (concatStringsSep "/" path);
       moduleDoc =
-        if builtins.length path >= 2 && lib.hasAttrByPath path nixvimInfo then
-          let
-            info = lib.getAttrFromPath path nixvimInfo;
-            maintainers = lib.unique (options.config.meta.maintainers.${info.file} or [ ]);
-            maintainersNames = builtins.map (m: m.name) maintainers;
-          in
+        let
+          info = optionalAttrs (hasAttrByPath path nixvimInfo) (getAttrFromPath path nixvimInfo);
+          maintainers = lib.unique (options.config.meta.maintainers.${info.file} or [ ]);
+          maintainersNames = builtins.map (m: m.name) maintainers;
+        in
+        # Validate this actually is an info attrset
+        if info ? file && info ? description && info ? url then
```

Comment:
without this you encountered false positives ?

Reply:
Yes, because the _final_ nixvimInfo contains nested attrs, each step along the path will check if it is an info set.